### ISSUE TITLE
[SB8]: JS error for stories without text

### ISSUE THREAD

**Issue by Jura** [undp/design-system#1359]
error `Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'languages')` thrown for stories without textual component: https://develop.design-system.pages.dev/?path=/story/foundation-logos--pnud-logo-blue

**Comment by CalebLuster** [comment]
Should be fixed now

**Comment by Jura** [comment]
@CalebLuster , I don't think so. It is probably an issue for other stories as well: https://post-sb8-upgrade-fixes.design-system.pages.dev/?path=/docs/foundation-typography-links--docs. Please scan through them to make sure the same treatment applied everywhere

**Comment by CalebLuster** [comment]
Just made a commit to fix this, I forgot to reference the ticket in the commit but here is a link to it

https://github.com/undp/design-system/commit/e47cf0ccdba8fe4d9119221dd201de29a976a0e8

**Comment by Jura** [comment]
Yep, looks like it is gone, closing
